components/eamxx/cime_config/eamxx_buildnml.py

```diff
@@ -745,6 +754,8 @@ def create_input_data_list_file(caseroot):
 ###############################################################################
 def do_cime_vars_on_yaml_output_files(case, caseroot):
 ###############################################################################
+    from yaml_utils import array_constructor
```

Comment:
I'd prefer this be at the top with the other imports.

Reply:
ok